pkg/storage/stores/shipper/compactor/retention/retention.go

```diff
@@ -214,6 +224,117 @@ func (s *Sweeper) Start() {
 	})
 }
 
+func getUserIDFromChunkID(chunkID string) (string, error) {
+	parts := strings.Split(chunkID, "/")
```

Comment:
I have a utility that splits without allocating string. Not super important but could be nice to avoid allocations in there in case we have a LOT of chunks to delete.

Reply:
Can you please share where I can find this utility? I looked it up but couldn't find it.